src/components/Listing/index.js

```diff
@@ -29,15 +29,15 @@ export default function Listing (props) {
   return (
     <ListGroup.Item>
       <p>{symbolWrapper} {description}</p>
-      <p className="mb-0"><span className="text-secondary">Category:</span> {category}</p>
+      <p className="mb-0"><span className="text-secondary">Category:</span> {displayTitle}</p>
       <p className="mb-0"><span className="text-secondary">Location:</span> {location}</p>
     </ListGroup.Item>
```

Comment:
Just some food for thought. This looks like a pretty good case for some semantic HTML.

```html
<section>
  <p>...</p>
  <h3>...</h3>
  <p>...</p>
</section>
```

I'm not 100% sure on the best markup, for example, it might be that you're supposed to have the heading as the first tag in a section, otherwise, you should used `aria-labelledby` on the section linking it to an id on the heading. Not a huge deal, just mentioning it because I saw the opportunity.